packages/web/src/components/user-name-and-badges/UserNameAndBadges.tsx

```diff
@@ -32,7 +34,7 @@ type UserNameAndBadgesProps =
   | UserNameAndBadgesWithIdProps
 
 const UserNameAndBadgesImpl = (props: UserNameAndBadgesImplProps) => {
-  const { user, onNavigateAway, classes } = props
+  const { user, onNavigateAway, classes, renderName } = props
```

Comment:
what about just using `user-link`?

Reply:
Oh nice yes I didn't know that was a thing